gammapy/irf/tests/test_irf_write.py

```diff
@@ -9,18 +9,17 @@
 
 class TestIRFWrite:
     def setup(self):
-        self.energy_lo = np.logspace(0, 1, 11)[:-1] * u.TeV
```

Comment:
Is there a good reason why the number of bins changed here? If not please undo...

Reply:
We had 10 bins each in energy, lon, lat - the one ambiguous case we decided not to support.
So, I changed it to rather have 9 bins in energy.

Reply:
Indeed, thanks!